packages/office-ui-fabric-react/src/components/ResizeGroup/ResizeGroup.Props.ts

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { ResizeGroup } from './ResizeGroup';
+
+export interface IResizeGroup {
+  measure?: () => void;
+}
+
+export interface IResizeGroupProps extends React.HTMLProps<ResizeGroup | HTMLElement> {
+
+  /**
+   * Optional callback to access the IResizeGroup interface. Use this instead of ref for accessing
+   * the public methods and properties of the component.
+   */
+  componentRef?: (component: IResizeGroup) => void;
```

Comment:
Oh. cDU.. I see.. was that always there? :) Then I'd be fine with removing this.